From 1429ad62bef1098f1cec26447cdc12934879a23c Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Fri, 4 Nov 2016 14:10:09 +0800
Subject: [PATCH] Revert "ARM64: cpufreq_sched: implement event
 CPUFREQ_GOV_LIMIT for governor"

This reverts commit d94634b0ab1bd61cc4088de63608287938a8fec2.

Fixed by commit 24884e54340e35d43bd09af0b12caef57a63458f
("sched/cpufreq_sched: Consolidated update")

Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 kernel/sched/cpufreq_sched.c | 16 +---------------
 1 file changed, 1 insertion(+), 15 deletions(-)

diff --git a/kernel/sched/cpufreq_sched.c b/kernel/sched/cpufreq_sched.c
index c608a9265d6a..e1d208e101ed 100644
--- a/kernel/sched/cpufreq_sched.c
+++ b/kernel/sched/cpufreq_sched.c
@@ -327,20 +327,6 @@ static int cpufreq_sched_stop(struct cpufreq_policy *policy)
 	return 0;
 }
 
-static int cpufreq_sched_limits(struct cpufreq_policy *policy)
-{
-	if (policy->max < policy->cur)
-		__cpufreq_driver_target(policy,
-					policy->max,
-					CPUFREQ_RELATION_H);
-	else if (policy->min > policy->cur)
-		__cpufreq_driver_target(policy,
-					policy->min,
-					CPUFREQ_RELATION_L);
-
-	return 0;
-}
-
 static int cpufreq_sched_setup(struct cpufreq_policy *policy,
 			       unsigned int event)
 {
@@ -354,7 +340,7 @@ static int cpufreq_sched_setup(struct cpufreq_policy *policy,
 	case CPUFREQ_GOV_STOP:
 		return cpufreq_sched_stop(policy);
 	case CPUFREQ_GOV_LIMITS:
-		return cpufreq_sched_limits(policy);
+		break;
 	}
 	return 0;
 }
-- 
2.35.3

